eclass/distutils-r1.eclass

```diff
@@ -425,6 +401,9 @@ unset -f _distutils_set_globals
 # on the sources directly, prepending setup.py arguments with
 # 'build --build-base ${BUILD_DIR}' to enforce keeping & using built
 # files in the specific root.
+#
+# In-source builds are deprecated and no longer supported in PEP517
+# mode.
```

Comment:
Very minor point, it is "PEP 517 mode" (with a space) in the documentation of `DISTUTILS_USE_PEP517` above. Maybe this should be consistent within the eclass?